StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedViewRepresentable.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  EmbeddedViewRepresentable.swift
+//  StripePaymentSheet
+//
+//  Created by Nick Porter on 1/30/25.
+//
+
+import SwiftUI
+@_spi(STP) import StripeCore
+
+struct EmbeddedViewRepresentable: UIViewRepresentable {
+    @ObservedObject var viewModel: EmbeddedPaymentElementViewModel
+
+    init(viewModel: EmbeddedPaymentElementViewModel) {
+        self.viewModel = viewModel
+        STPAnalyticsClient.sharedClient.addClass(toProductUsageIfNecessary: EmbeddedSwiftUIProduct.self)
```

Comment:
Why add it here vs the `EmbeddedPaymentElementViewModel` init?

Reply:
I _very_ slightly prefer to put this here, since it's an internal class, and keep the public EmbeddedPaymentElementViewModel init empty. In an effort to keep our public classes a bit cleaner in case anyone is looking through them, kinda like EmbeddedPaymentElement+Internal motivation.

Reply:
Hmm, I see, I'll move it.